kubernetes-repositories.md

```diff
@@ -73,6 +73,34 @@ Create a broader base of repositories than the existing gh/kubernetes/kubernetes
    * All OWNERS must be members of standing as defined by ability to vote in Kubernetes steering committee elections. in the Kubernetes community
    * Repository must be approved by SIG-Architecture
 
+### Removing Repositories
+
+As important as it is to add new repositories, it is equally important to
+prune old repositories that are no longer relevant or useful.
+
+It is in the best interests of everyone involved in the Kubernetes community
+that our various projects and repositories are active and healthy. This
+ensures that repositories are kept up to date with the latest Kubernetes
+wide processes, it ensures a rapid response to potential required fixes
+(e.g. critical security problems) and (most importantly) it ensures that
+contributors and users receive quick feedback on their issues and 
+contributions.
+
+#### Grounds for removal
+A repository may be removed from the project if it is _not_ an associated
+repository and any of the following criteria are met.
+
+   * There are no longer any active maintainers for the project and no 
+replacements can be found.
+   * PRs or Issues have gone un-addressed for longer than six months.
+   * There have been no new commits or other chagnes in more than a year.
+
+
+#### Procedure for removal
+When a repository is set for removal, it is moved into the [kubernetes-retired](https://github.com/kubernetes-retired) organization. This maintains the
+complete record of issues, PRs and other contributions, but makes it clear
+that the repository should be considered archival, not active. 
```

Comment:
Should we also use the github archive feature?

Reply:
added.